ux.symfony.com/templates/ux_packages/translator.html.twig

```diff
@@ -0,0 +1,251 @@
+{% extends 'packageBase.html.twig' %}
+
+{% block component_header %}
+    {% component PackageHeader with {
+        package: 'translator',
+        eyebrowText: 'eyebrow text'
+    } %}
+        {% block title_header %}
+            title
+            <span class="playfair ps-2">header</span>
+        {% endblock %}
+
+        {% block sub_content %}
+            sub_content
```

Comment:
Write & manage your translations like you always do. Then use them just as easily in JS.